OptionSelector/ListTableViewController.m

```diff
@@ -0,0 +1,82 @@
+//
+//  ListTableViewController.m
+//  OptionSelector
+//
+//  Created by Justine Gartner on 8/11/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "ListTableViewController.h"
+
+@interface ListTableViewController ()
+
+@end
+
+@implementation ListTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+    self.title = self.basic.detailTitle;
+}
+
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
```

Comment:
You can leave this method out entirely if you're not using sections; the default value is 1.